### DISCUSSION TITLE
Improvements to Jira integration

### DISCUSSION THREAD

**Discussion by pascalrobert** [DefectDojo/django-DefectDojo#8365] [Ideas]
Hi,

I'm starting using DefectDojo, and I would like to make some changes to the Jira integration. Mainly:

* If the user who is creating the Jira issue from DejectDojo is also a Jira user, the user will become the reporter in Jira, if no default user is provider in the engagement or product.

* When the finding is coming from SARIF, convert the Markdown to the Jira Text Formatting Notation format, so that the formatting is looking good in Jira.

* The source file link would be in the Development panel (but I'm not sure it can be done without a commit id)

* All links in References will be links in the issue, instead of being in the description.

* Adjusting the documentation to say that to see the priority and due date coming from DefectDojo, those fields must be in the screen of the Bug issue type (in a team-managed project in Jira Cloud, they are not show by default).

* Show the status of the Jira issue in the finding. 

Any input on this?

**Comment by testaccount90009** [comment]
"Show the status of the Jira issue in the finding."  -- regarding this, when I select a Product, an Engagement, the Findings in that engagement, I can see "Status" as a field with the Jira link to it.  With the Jira integration, doesn't the Status maintain parity with whatever is reflected in Jira/Dojo?  I currently have a successful Jira integration and the Status of the Jira issue does maintain sync with each respective tool.  If I mark it as a False Positive, Resolved, or Accepted on either tool, it does reflect correctly in the other.  

Perhaps I have misunderstood the ask of this?

**Comment by testaccount90009** [comment]
"If the user who is creating the Jira issue from DefectDojo is also a Jira user, the user will become the reporter in Jira, if no default user is provider in the engagement or product." -- I'm not sure I understand this either, as the DefectDojo <-> Jira integration requires a Jira user that creates the tickets and that user is the Reporter by default already.
